src/utils/constants.h

```diff
@@ -19,6 +19,17 @@ const byte MENU_ITEM_END_OF_MENU = 8;
 const byte MENU_ITEM_LIST = 9;
 const byte MENU_ITEM_PROGRESS = 10;
 //
+// Control codes
+//
+#define BACKSPACE 8  // Backspace
```

Comment:
MacBook prints

```
#CMD# INPUT=127
#CMD# Call=127
```

for Backspace

Reply:
127 is DEL in ASCII, so it's also valid. Both 8 and 127 will be mapped to `BACKSPACE`. Will add it as well.